CHANGELOG.md

```diff
@@ -380,7 +380,7 @@
 
 ### Fixed
 * Comments in SynArgPats.NamePatPairs are lost. [#2541](https://github.com/fsprojects/fantomas/issues/2541)
-* Vanity alignment used inside base ctor call. [#2111](https://github.com/fsprojects/fantomas/issues/2111)
+* Whitespace alignment used inside base ctor call. [#2111](https://github.com/fsprojects/fantomas/issues/2111)
```

Comment:
I don't really like changing such a thing in the changelog file.
It feels too much like rewriting history to me.

Reply:
Mmh, I see, maybe we can leave the changelog if that's contentious and the other part of the change seems beneficial?